actor/src/main/resources/reference.conf

```diff
@@ -1244,6 +1244,9 @@ pekko {
     # Terminate the ActorSystem in the last phase actor-system-terminate.
     terminate-actor-system = on
 
+    # The timeout that will be used when calling .close on an ActorSystem
```

Comment:
add `since 1.3.0`

Reply:
Same as https://github.com/apache/pekko/pull/2486#discussion_r2521369096, this is not part of public API